src/app/scripts/cac/pages/cac-pages-home.js

```diff
@@ -201,7 +214,20 @@ CAC.Pages.Home = (function ($, ModeOptions,  MapControl, TripOptions, SearchPara
         }
 
         // load directions if origin and destination set
-        $(document).ready(directionsControl.setFromUserPreferences());
+        $(document).ready(loadInitialMethod);
+    }
+
+    function loadInitialMethod() {
+        if (!UserPreferences.isDefault('method')) {
+            var method = UserPreferences.getPreference('method');
```

Comment:
Just made reference to this bit of code in issue #715, but if it doesn't turn out to be useful for that I think you're right that it can go.